README.md

```diff
@@ -36,7 +36,11 @@ All contributors are required to abide by our [Code of Conduct](https://github.c
 ## All Contributors
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
+[![All Contributors](https://img.shields.io/badge/all_contributors-107-orange.svg?style=flat-square)](#contributors-)
+
 [![All Contributors](https://img.shields.io/badge/all_contributors-108-orange.svg?style=flat-square)](#contributors-)
```

Comment:
There are a couple of things here:

- There are duplication in the badge (line 40 & 42), which not what we want.
- The badge number should increase and not decrease. Currently, we have 108 contributors.